### PR TITLE
Feature/entitlement execute script

### PR DESCRIPTION
**Pull request by jcabrerizo**
Adds a new `Entitlement` to control the Groovy script execution and a new `EntitlementManager` with "**PowerUser**" with the same entitlements as root but the ability to execute groovy scripts

### CODE REVIEW

**Review by iuliana — COMMENTED** [review]
Looks good to me. And looks good to merge.

File: core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/Entitlements.java [hunk 1]
```diff
@@ -480,6 +503,8 @@ public static EntitlementManager load(@Nullable ManagementContext mgmt, Brooklyn
             return minimal();
         } else if ("user".equalsIgnoreCase(type)) {
             return user();
+        } else if ("powerUser".equalsIgnoreCase(type) || "power_user".equalsIgnoreCase(type)) {
+            return powerUser();
```
Comment (algairim) [inline core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/Entitlements.java:507]: Do we have constants for these to re-use?

**Review by iuliana — APPROVED** [review]
LGTM

File: core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/Entitlements.java (hunk 1 above)
Comment (jcabrerizo) [reply core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/Entitlements.java:511]: No for now. If we need to reuse the name of the entitlement manager we can create, but if not, I think is more clear to see directly the value here.

**Pull request merged**